src/services/docGenService.ts

```diff
@@ -65,7 +67,7 @@ export class DocGenService {
       }
       const enableNewDocsPanel = workspace
         .getConfiguration("dbt")
-        .get<boolean>("enableNewDocsPanel", false);
```

Comment:
I thought we agreed to get rid of this configuration

Reply:
We will remove this config in next release. in the first version, we will make new doc gen as default and remove switching button. but user can still go to settings and enable legacy. In next version we will remove this config